src/questions-21-40/q33.py

```diff
@@ -7,5 +7,29 @@
 # less than one in value, and containing two digits in the numerator and denominator.
 #
 # If the product of these four fractions is given in its lowest common terms, find the value of the denominator.
+import functools
+import math
+from operator import mul
+
+
+def get_curious_fractions():
+    temps = [(num, denum)
+            for num in range(1, 99)
+                for denum in range(num+1, 100)
+                    if (num % 10 == denum // 10
+                        and denum % 10 != 0
+                        and (num // 10) / (denum % 10) == num / denum)]
+    nums, denums = zip(*temps)
+    return nums, denums
+
+
+nums, denums = get_curious_fractions()
+
+numerators = functools.reduce(mul, nums)
+denomenators = functools.reduce(mul, denums)
+
+print(denomenators/math.gcd(numerators, denomenators))
```

Comment:
:D nice code mate! I didn't know this was what it wanted

Reply:
Yea, I figure it was really this weird sentences getting you confused. Seem like an easy question relative to others you're doing these days.

Reply:
maybe for your mathematical mind haha